deps/build.jl

```diff
@@ -9,46 +9,66 @@ end
 
 function write_depsfile(path)
     open(depsfile,"w") do f
-        print(f,"const libcplex = ")
+        print(f, "const libcplex = ")
         show(f, path) # print with backslashes excaped on windows
         println(f)
     end
 end
 
-@static if (VERSION >= v"0.7.0-DEV.3382" && Sys.isapple()) || (VERSION < v"0.7.0-DEV.3382" && is_apple())
-    Libdl.dlopen("libstdc++",Libdl.RTLD_GLOBAL)
+@static if Sys.isapple()
```

Comment:
Thanks for the cleanup!

Reply:
Shall I make a PR with similar changes throughout the package? (The question being: is there a rewrite effort in progress, like for Gurobi and GLPK?)